diff --git a/modules/wechat_qrcode/src/wechat_qrcode.cpp b/modules/wechat_qrcode/src/wechat_qrcode.cpp
index 64aad736..c211c33a 100644
--- a/modules/wechat_qrcode/src/wechat_qrcode.cpp
+++ b/modules/wechat_qrcode/src/wechat_qrcode.cpp
@@ -55,8 +55,6 @@ WeChatQRCode::WeChatQRCode(const String& detector_prototxt_path,
     if (!detector_caffe_model_path.empty() && !detector_prototxt_path.empty()) {
         // initialize detector model (caffe)
         p->use_nn_detector_ = true;
-        CV_Assert(utils::fs::exists(detector_prototxt_path));
-        CV_Assert(utils::fs::exists(detector_caffe_model_path));
         p->detector_ = make_shared<SSDDetector>();
         auto ret = p->detector_->init(detector_prototxt_path, detector_caffe_model_path);
         CV_Assert(ret == 0);
@@ -71,8 +69,6 @@ WeChatQRCode::WeChatQRCode(const String& detector_prototxt_path,
     if (!super_resolution_prototxt_path.empty() && !super_resolution_caffe_model_path.empty()) {
         p->use_nn_sr_ = true;
         // initialize dnn model (caffe format)
-        CV_Assert(utils::fs::exists(super_resolution_prototxt_path));
-        CV_Assert(utils::fs::exists(super_resolution_caffe_model_path));
         auto ret = p->super_resolution_model_->init(super_resolution_prototxt_path,
                                                     super_resolution_caffe_model_path);
         CV_Assert(ret == 0);
